docs/web/export-services.py

```
@@ -0,0 +1,140 @@
+def get_git_root() -> str:
+    """Get the git repository root directory.
+
+    Returns:
+        str: The absolute path to the git repository root directory.
+
+    Raises:
+        RuntimeError: If git executable is not found or not in a git repository.
+    """
+    git_cmd = shutil.which("git")
+    if git_cmd is None:
+        raise RuntimeError("Git not found on PATH") from None
```

Ruff (0.14.6)

Avoid specifying long messages outside the exception class

(TRY003)